.travis.yml

```diff
@@ -28,12 +28,12 @@ before_install:
 - cabal --version
 
 install:
-- cabal new-update -v hackage.haskell.org
-- cabal new-configure --enable-tests --write-ghc-environment-files=always
-- cabal new-build --only-dependencies -j
+- cabal new-update -v
+- cabal new-configure --enable-tests --disable-optimization --write-ghc-environment-files=always --jobs=2
+- cabal new-build --only-dependencies
 
 script:
-- cabal new-build -j
+- cabal new-build
```

Comment:
Moving that to the `new-configure` step is a great idea. We should do this in `fused-effects` too.